src/main/java/sysc4806/project24/mini_survey_monkey/models/MultipleChoiceQuestion.java

```diff
@@ -0,0 +1,21 @@
+package sysc4806.project24.mini_survey_monkey.models;
+
+import jakarta.persistence.*;
+import java.util.List;
+
+@Entity
+public class MultipleChoiceQuestion extends Question {
+
+    @ElementCollection
+    @CollectionTable(name = "multiple_choice_options", joinColumns = @JoinColumn(name = "question_id"))
```

Comment:
Do you think it would make sense to use only one table for multiple choice options where each row is a different option?

Reply:
No if you think this is a good idea that's fine! Just make sure you update the DB schema to match please.